connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -491,6 +490,10 @@ private void closePartitions() {
     private class HandleRebalance implements ConsumerRebalanceListener {
         @Override
         public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
+            // Increase the commit sequence number so that asynchronous commits that are not complete
+            // won't overwrite what we do here
+            commitSeqno += 1;
+            committing = false;
```

Comment:
Hmm... Is it actually necessary? In the revocation callback, we trigger a synchronous commit which will bump the sequence number. Isn't that sufficient to fence any late commit callbacks?